language/phpmailer.lang-id.php

```diff
@@ -8,20 +8,20 @@
 
 $PHPMAILER_LANG['authenticate']         = 'Kesalahan SMTP: Tidak dapat mengotentikasi.';
 $PHPMAILER_LANG['connect_host']         = 'Kesalahan SMTP: Tidak dapat terhubung ke host SMTP.';
-$PHPMAILER_LANG['data_not_accepted']    = 'Kesalahan SMTP: Data tidak diterima peladen.';
+$PHPMAILER_LANG['data_not_accepted']    = 'Kesalahan SMTP: data tidak diterima.';
```

Comment:
Was removing capitalisation of `Data` intentional there? Other messages of a similar type do capitalise that way (e.g. `file_open`).